runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -160,7 +183,7 @@ public void onPlayerLootReceived(final PlayerLootReceived playerLootReceived)
 		final String name = player.getName();
 		final int combat = player.getCombatLevel();
 		final LootTrackerItemEntry[] entries = buildEntries(stack(items));
-		SwingUtilities.invokeLater(() -> panel.addLog(name, combat, entries));
+		SwingUtilities.invokeLater(() -> panel.addEntry(name, combat, entries, config.groupLoot()));
```

Comment:
I would just do setGroupLoot on the panel from configChanged instead of adding new method param

Reply:
I was trying to avoid having that sort of stuff in the UI classes

Reply:
I dont mean of you adding configChanged even to panel. But to make setter and set it on panel in configChanged event from plugin. I dont see how adding parameter to multiple methods for param that is basically global is better.